lib/astutils.cpp

```diff
@@ -3099,7 +3099,7 @@ namespace {
         template<class F>
         const Token* operator()(const Token* start, const Token* end, F f) const
         {
-            return findToken(start, end, f);
+            return findToken(start, end, std::move(f));
```

Comment:
The predicate is taken as const ref, so this will cause an extra copy.

Reply:
Good catch. Leftover from some wrongly applied changes.

Reply:
I wonder why clang-tidy did not report this. Possibly because it is not yet a `std::function` and still a function pointer. Something to look into.